packages/react/src/date-picker/DatePickerInput.tsx

```diff
@@ -16,6 +15,7 @@ export type DateInputProps = Omit<TextInputProps, 'invalid'> & {
 	};
 	buttonRef: RefObject<HTMLButtonElement>;
 	buttonOnClick: MouseEventHandler<HTMLButtonElement>;
+	buttonAriaLabel: string;
```

Comment:
This has been moved into a prop, so date-picker and date-range-picker get generate their own custom aria-label